I14380: Fixed a couple of things in the 'proc_configuring-a-workspace… (#799)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc

.Prerequisites

> * An OKD, OCP, or OSD cluster, if using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift] (OKD). Or, you can use a Kubernetes cluster such as link:https://github.com/kubernetes/minikube#installation[Minikube].

Reviewer: ```suggestion
* An OKD, OCP, or OSD cluster for using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift] (OKD). Or, use a Kubernetes cluster, such as link:https://github.com/kubernetes/minikube#installation[Minikube].
```

. Commit a `devfile.yaml` file into the root of your Git repository.

> . When the repository contains a `devfile.yaml` file, point a running Che 7 instance at that project, and it will use the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the `/f?url=` API.

Reviewer: ```suggestion
. When the repository contains a `devfile.yaml` file, point a running Che 7 instance at that project. The Che 7 instance then uses the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the `/f?url=` API.
```

* For example, when using link:{site-baseurl}che-7/running-che-locally/#deploying-che-using-chectl[Che inside Minishift]: `++http://che-che.<IP-address>.nip.io/f?url=https://github.com/eclipse/che++`.
+
image::workspaces/che-in-che-devfile.png[Che in Che]
+
* Or, point at a plain text file: `++http://che-che.<IP-address>.nip.io/f?url=https://gist.githubusercontent.com/nickboldt/9551284c2c779a93ccf16bc801416205/raw/7ad1e949880d7f8ed30360e74bb9f8426450db05/devfile.yaml++`.

> * If you are familiar with link:https://github.com/che-incubator/chectl/[chectl], reference a local devfile. For example, to create a workspace in which to build the Che project:

Reviewer: ```suggestion
* Alternatively, use link:https://github.com/che-incubator/chectl/[chectl] to reference a local devfile. For example, to create a workspace in which to build the Che project:
```
